sunkit_pyvista/plotter.py

```diff
@@ -237,7 +237,7 @@ def plot_map(self, m, clip_interval: u.percent = None, **kwargs):
         else:
             clim = [0, 1]
         cmap = self._get_cmap(kwargs, m)
-        self.plotter.add_mesh(map_mesh, cmap=cmap, clim=clim, **kwargs)
+        self.plotter.add_mesh(map_mesh, cmap=cmap, clim=clim, show_scalar_bar=False, **kwargs)
```

Comment:
Instead we should do `kwargs.setdefault('show_scalar_bar', False)` so users can still turn on the scalar bar if they want to.

Reply:
Right, I thought we didn't need this functionality. I'll make this change :smile: